packages/code-studio/src/styleguide/SamplesMenu.tsx

```diff
@@ -56,7 +56,7 @@ export function SamplesMenu(): JSX.Element {
     );
 
     const spectrumComparisonSamples = document.querySelector(
-      `#${SPECTRUM_COMPARISON_SAMPLES_ID}`
+      `#sample-section-${SPECTRUM_COMPARISON_SAMPLES_ID}`
```

Comment:
This was a bug causing duplicate ids and the menu would always navigate to the first occurrence